<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileManagerPage</class>
 <widget class="QWidget" name="FileManagerPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Êñá‰ª∂ÁÆ°ÁêÜÂô®</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="topMargin">
    <number>15</number>
   </property>
   <property name="rightMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   
   <!-- Ê†áÈ¢ò -->
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>üìÅ Êñá‰ª∂ÁÆ°ÁêÜÂô®</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font-size: 20px; font-weight: bold; color: #2c3e50;</string>
     </property>
    </widget>
   </item>
   
   <!-- Â∑•ÂÖ∑Ê†è -->
   <item>
    <layout class="QHBoxLayout" name="toolbarLayout">
     <item>
      <widget class="QPushButton" name="upButton">
       <property name="text">
        <string>‚¨Ü ‰∏äÁ∫ß</string>
       </property>
       <property name="toolTip">
        <string>ËøîÂõû‰∏äÁ∫ßÁõÆÂΩï</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="homeButton">
       <property name="text">
        <string>üè† ‰∏ªÈ°µ</string>
       </property>
       <property name="toolTip">
        <string>Ë∑≥ËΩ¨Âà∞Áî®Êà∑ÁõÆÂΩï</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refreshButton">
       <property name="text">
        <string>üîÑ Âà∑Êñ∞</string>
       </property>
       <property name="toolTip">
        <string>Âà∑Êñ∞ÂΩìÂâçÁõÆÂΩï</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pathEdit">
       <property name="placeholderText">
        <string>ËæìÂÖ•Ë∑ØÂæÑÂπ∂ÂõûËΩ¶Ë∑≥ËΩ¨...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   
   <!-- ÊêúÁ¥¢ÂíåËøáÊª§ -->
   <item>
    <layout class="QHBoxLayout" name="filterLayout">
     <item>
      <widget class="QLineEdit" name="searchEdit">
       <property name="placeholderText">
        <string>üîç ÊêúÁ¥¢Êñá‰ª∂...</string>
       </property>
       <property name="minimumWidth">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="filterCombo">
       <property name="minimumWidth">
        <number>150</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="filterSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   
   <!-- ‰∏ªÂÜÖÂÆπÂå∫ -->
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     
     <!-- Â∑¶‰æßÊ†ëÂΩ¢ÁõÆÂΩï -->
     <item>
      <widget class="QGroupBox" name="navGroup">
       <property name="title">
        <string>ÁõÆÂΩï</string>
       </property>
       <property name="minimumWidth">
        <number>200</number>
       </property>
       <property name="maximumWidth">
        <number>250</number>
       </property>
       <layout class="QVBoxLayout" name="navLayout">
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="headerHidden">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     
     <!-- ‰∏≠Èó¥Êñá‰ª∂ÂàóË°® -->
     <item>
      <widget class="QGroupBox" name="filesGroup">
       <property name="title">
        <string>Êñá‰ª∂</string>
       </property>
       <layout class="QVBoxLayout" name="filesLayout">
        <item>
         <widget class="QListView" name="listView">
          <property name="iconSize">
           <size>
            <width>48</width>
            <height>48</height>
           </size>
          </property>
          <property name="spacing">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     
     <!-- Âè≥‰æßÊñá‰ª∂‰ø°ÊÅØ -->
     <item>
      <widget class="QGroupBox" name="infoGroup">
       <property name="title">
        <string>Êñá‰ª∂‰ø°ÊÅØ</string>
       </property>
       <property name="minimumWidth">
        <number>200</number>
       </property>
       <property name="maximumWidth">
        <number>250</number>
       </property>
       <layout class="QVBoxLayout" name="infoLayout">
        <item>
         <widget class="QLabel" name="fileNameLabel">
          <property name="text">
           <string>ÂêçÁß∞: --</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileSizeLabel">
          <property name="text">
           <string>Â§ßÂ∞è: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileTypeLabel">
          <property name="text">
           <string>Á±ªÂûã: --</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="fileDateLabel">
          <property name="text">
           <string>‰øÆÊîπ: --</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="infoSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   
   <!-- Áä∂ÊÄÅÊ†è -->
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string>Â∞±Áª™</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #7f8c8d; padding: 5px;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
